app/src/main/java/com/infomaniak/mail/data/cache/mailboxContent/RefreshController.kt

```diff
@@ -401,10 +401,14 @@ class RefreshController @Inject constructor(
         return impactedThreads
     }
 
+    private fun List<Int>.getAtStart(size: Int) = subList(0, size) to subList(size, count())
+
+    private fun List<Int>.getAtEnd(size: Int) = subList(count() - size, count()) to subList(0, count() - size)
+
```

Comment:
Maybe `amount` or `quantity` over `size`? `size` feels a bit vague but more importantly it's already a reserved name in the context of a list, so it can be more misleading than one of the alternatives? What do you think?